file_picker_web/lib/file_picker_web.dart

```diff
@@ -1,9 +1,34 @@
 import 'dart:async';
-
 import 'package:file_picker_platform_interface/file_picker_platform_interface.dart';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'dart:html' as html;
 
+
+/// File Object wrapper 
+/// 
+/// [file]  `html.File` object that contains picked file
+class File {
```

Comment:
I'm not sure if naming this to `File` would confuse some users as it's just wrapping a `html.File`. What advantages would you see with this?